besu/src/main/resources/log4j2.xml

```diff
@@ -1,16 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="INFO">
   <Properties>
-    <Property name="root.log.level">INFO</Property>
+    <Property name="root.log.level">${env:LOG_LEVEL:-INFO}</Property>
+    <Property name="root.log.logger">${env:LOGGER:-Console}</Property>
+    <Property name="host">${env:HOST:-${docker:containerId:-localhost}}</Property>
+    <Property name="splunk.url">${env:SPLUNK_URL}</Property>
+    <Property name="splunk.token">${env:SPLUNK_TOKEN}</Property>
+    <Property name="splunk.index">${env:SPLUNK_INDEX}</Property>
+    <Property name="splunk.source">${env:SPLUNK_SOURCE:-besu}</Property>
+    <Property name="splunk.sourcetype">${env:SPLUNK_SOURCETYPE:-besu}</Property>
+    <Property name="splunk.batch_size_bytes">${env:SPLUNK_BATCH_SIZE_BYTES:-10000}</Property>
+    <Property name="splunk.batch_size_count">${env:SPLUNK_BATCH_SIZE_COUNT:-10}</Property>
```

Comment:
10 seem quite low for a default batch size.

Reply:
I made up those numbers. Does 100 sound good?

Reply:
I pushed to 100 for now.

Reply:
Added suggestion for new defaults.